### ISSUE TITLE
Fans will only sync to GPU 1 of 2

### ISSUE THREAD

**Issue by smbell1979** [Rem0o/FanControl.Releases#118]
I'm trying to sync my case fans to my second GPU, so I've created a sync curve for this GPU and selected it for the rear fan's curve selection. It only seems to want to sync to the first GPU fan speed even though the curve is not set to that GPU as the selected control.

All the fan speeds and everything else appear to be working correctly.

I'm running 1080Ti's on a Asus Crosshair VIII Hero (Win10 20H2) with a 5950x.

**Comment by Rem0o** [comment]
I'm having a hard time figuring out your issue, could you explain with screenshots?

**Comment by smbell1979** [comment]
Sure, I illustrated a screenshot that hopefully will help.
![Screenshot_2_markup](https://user-images.githubusercontent.com/76536166/104662536-e94e7300-567f-11eb-8e8f-2f14d3490e85.jpg)

**Comment by Rem0o** [comment]
Could you expand all the control cards also?

**Comment by smbell1979** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/76536166/104662789-64178e00-5680-11eb-9f47-6f8f2f4b9b59.jpg)

**Comment by Rem0o** [comment]
Yup, from your previous screenshot, seems like it's bound to 1080 TI 1.

**Comment by smbell1979** [comment]
It doesn't seem to want to save the correct way when I save the config and reload the app, seems like it just isn't getting set. Even if I change it to 2, it never actually works.

**Comment by Rem0o** [comment]
Just for fun, could you rename your 2 1080 ti controls to like : "Hello" and "Darling", and try to set the Sync with "Darling" and see if it works?

**Comment by smbell1979** [comment]
Negative, same behavior.

**Comment by Rem0o** [comment]
If you enable 1 of the GPU control and apply a flat curve, does both follow it?

**Comment by smbell1979** [comment]
Everything works as it should with each GPU separately except the sync curve.

I've been poking around in the userConfig.json file this morning and actually got it to semi-work with a caveat.

I created a new config with just a single sync curve and let it build the file. I tried modifying values one step at a time and watched the changes in the program. 

Since there were two identically named controls for the GPUs I renamed the second one. **didn't work**

I tried to set the sync to the second control. **didn't work**

I changed the values of the first control to use the second GPUs sensor identifiers **didn't work**

I removed all traces of the first GPU control and GPU sensors and set the sync to the remaining second GPU control. When I started it up, it complained that it couldn't find the control with a "missing control" error even though it was there in the file. I looked in the sync curve drop down box and the control was actually listed and when I set the sync to this **it worked**. 

What I noticed though was that the first removed GPU control was automatically created every time I opened the app no matter what. Setting the sync to this didn't do anything, but it was there. 

I don't know if any of this helps or not.
